src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -78,10 +78,13 @@ function useChatUtilsHook(): UseChatUtilsHook {
 
   const goToCommunityScreen = (channel: ChannelList) => {
     setChannelAsRead(channel);
-
     const topicName = channel?.name?.replace(/^#/, '');
     const navigationParam = {
-      id: convertTopicNameToTopicPageScreenParam(topicName)
+      id: convertTopicNameToTopicPageScreenParam(topicName),
+      channelPicture: channel.channelPicture,
+      coverImage: channel.rawJson.cover_image,
+      isFollowing: channel.rawJson.is_following,
+      memberCount: channel.rawJson.topic_follower_count
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code is trying to access properties like `channelPicture`, `rawJson.cover_image`, `rawJson.is_following`, and `rawJson.topic_follower_count` directly from the `channel` object. This could potentially lead to a runtime error if any of these nested properties do not exist. It would be safer to use optional chaining when accessing these properties.
```diff
-      const navigationParam = {
-       id: convertTopicNameToTopicPageScreenParam(topicName),
-       channelPicture: channel.channelPicture,
-       coverImage: channel.rawJson.cover_image,
-       isFollowing: channel.rawJson.is_following,
-       memberCount: channel.rawJson.topic_follower_count
-     };
+      const navigationParam = {
+       id: convertTopicNameToTopicPageScreenParam(topicName),
+       channelPicture: channel?.channelPicture,
+       coverImage: channel?.rawJson?.cover_image,
+       isFollowing: channel?.rawJson?.is_following,
+       memberCount: channel?.rawJson?.topic_follower_count
+     };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->